.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "printWidth": 120,
```

Comment:
I usually go with 80 chars...maybe I'm getting old :rofl: 

I guess this setting is not necessarily a hard limit though, from what I understand from the docs.
https://prettier.io/docs/en/options.html

Reply:
> I usually go with 80 chars...maybe I'm getting old
lol okay boomer

but yea 80 chars is definitely the default, and i think there's a case to be made for forcing lines to be concise. but given the amount of screen real estate folks have these days...and the [progress of technology](https://softwareengineering.stackexchange.com/a/148678)...i find sometimes that single expressions will get split across multiple lines in a way that makes them hard to read.

That being said, since datafruits is mostly just defining a bunch of classes with some methods, there's not really too many super long lines, and there's def a case to be made for 80 chars. I don't really have a preference one way or the other -- except for single quotes. i love single quotes.

Reply:
Ok no big deal really, I may open a PR to change it later if I feel strongly about it.